integration_tests/publish_intent_requests_test.go

```diff
@@ -0,0 +1,153 @@
+		contentItemPayload = []byte(contentItemJSON)
+		urlArbiterResponse = `{"path":"/vat-rates","publishing_app":"mainstream_publisher"}`
+
+		testPublishingAPI                *httptest.Server
+		testURLArbiter, testContentStore *ghttp.Server
+
+		urlArbiterResponseCode, contentStoreResponseCode           int
+		urlArbiterResponseBody, contentStoreResponseBody, endpoint string
+
+		expectedResponse HTTPTestResponse
+	)
+
+	BeforeEach(func() {
+		TestRequestOrderTracker = make(chan TestRequestLabel, 2)
+
+		testURLArbiter = ghttp.NewServer()
+		testContentStore = ghttp.NewServer()
+
+		testPublishingAPI = httptest.NewServer(main.BuildHTTPMux(testURLArbiter.URL(), testContentStore.URL()))
+		endpoint = testPublishingAPI.URL + "/publish-intent/vat-rates"
+	})
+
+	AfterEach(func() {
+		testURLArbiter.Close()
+		testContentStore.Close()
+		testPublishingAPI.Close()
+		close(TestRequestOrderTracker)
+	})
+
+	Describe("/publish-intent", func() {
+		Context("PUT", func() {
+			BeforeEach(func() {
+				testURLArbiter.AppendHandlers(ghttp.CombineHandlers(
+					trackRequest(URLArbiterRequestLabel),
+					ghttp.VerifyRequest("PUT", "/paths/vat-rates"),
+					ghttp.VerifyJSON(`{"publishing_app": "mainstream_publisher"}`),
+					ghttp.RespondWithPtr(&urlArbiterResponseCode, &urlArbiterResponseBody),
+				))
+
+				testContentStore.AppendHandlers(
+					ghttp.CombineHandlers(
+						trackRequest(ContentStoreRequestLabel),
+						ghttp.VerifyRequest("PUT", "/publish-intent/vat-rates"),
+						ghttp.VerifyJSON(contentItemJSON),
+						ghttp.RespondWithPtr(&contentStoreResponseCode, &contentStoreResponseBody),
+					),
+				)
+			})
+
+			Context("when URL arbiter errs", func() {
+				It("returns a 422 status with the original response", func() {
+					urlArbiterResponseCode = 422
+					urlArbiterResponseBody = `{"path":"/vat-rates","publishing_app":"mainstream_publisher","errors":{"base_path":["is not valid"]}}`
+
+					actualResponse := doRequest("PUT", endpoint, contentItemPayload)
+
+					Expect(testURLArbiter.ReceivedRequests()).To(HaveLen(1))
+					Expect(testContentStore.ReceivedRequests()).To(BeEmpty())
+
+					expectedResponse = HTTPTestResponse{Code: 422, Body: urlArbiterResponseBody}
+					assertSameResponse(actualResponse, &expectedResponse)
+				})
+
+				It("returns a 409 status with the original response", func() {
+					urlArbiterResponseCode = 409
+					urlArbiterResponseBody = `{"path":"/vat-rates","publishing_app":"mainstream_publisher","errors":{"base_path":["is already taken"]}}`
+
+					actualResponse := doRequest("PUT", endpoint, contentItemPayload)
+
+					Expect(testURLArbiter.ReceivedRequests()).To(HaveLen(1))
+					Expect(testContentStore.ReceivedRequests()).To(BeEmpty())
+
+					expectedResponse = HTTPTestResponse{Code: 409, Body: urlArbiterResponseBody}
+					assertSameResponse(actualResponse, &expectedResponse)
+				})
+			})
+
+			It("registers a path with URL arbiter and then publishes the content to the content store", func() {
```

Comment:
`then publishes the content` maybe it should be `then pushes the publish intent to content store`?